src/proxy-ui-api/frontend/tests/e2e/pages/tabs/settingsTab.js

```diff
@@ -118,9 +156,69 @@ const settingsTab = {
   sections: {
     systemParametersTab: {
       selector:
-        '//div[contains(@class, "v-tabs-bar__content")]//a[@data-test="system"]',
+        '//div[.//a[contains(@class, "v-tab--active") and @data-test="settings"]]//div[contains(@class, "base-full-width")]',
```

Comment:
Consider using a corresponding data-test attribute with value `system-parameters-tab` instead of looking up a div with `base-full-width` css class.

Reply:
Added data-test attributes to actual code and used shorter and more descriptive locators where possible.